src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGoHome.java

```diff
@@ -39,6 +40,16 @@ public void startExecuting()
             return;
         }
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)